crates/kornia-tensor/src/tensor.rs

```diff
@@ -1374,6 +1397,15 @@ mod tests {
         Ok(())
     }
 
+    #[test]
+    fn from_raw_parts() -> Result<(), TensorError> {
+        let data: Vec<u8> = vec![1, 2, 3, 4];
```

Comment:
This buffer is freed up a the end of the scope.

Reply:
fixed